diff --git a/node_modules/strapi-paypal/admin/src/components/CreateProduct/index.js b/node_modules/strapi-paypal/admin/src/components/CreateProduct/index.js
index d455b4f..6de28c2 100644
--- a/node_modules/strapi-paypal/admin/src/components/CreateProduct/index.js
+++ b/node_modules/strapi-paypal/admin/src/components/CreateProduct/index.js
@@ -42,16 +42,14 @@ const CreateProduct = ({ isVisible, handleClose, handleClickSave }) => {
     productType: "",
   });
 
-  const handleChange = (event) => {
-    const { name, value } = event.target;
+  const handleTitleChange = (event) => {
+    const { value } = event.target;
+    setTitle(value);
+  };
 
-    if (name === "title") {
-      setTitle(value);
-      setError({ ...error, title: "" });
-    } else if (name === "description") {
-      setDescription(value);
-      setError({ ...error, description: "" });
-    }
+  const handleDescriptionChange = (event) => {
+    const { value } = event.target;
+    setDescription(value);
   };
 
   const handleChangePaymentType = (value) => {
@@ -235,7 +233,7 @@ const CreateProduct = ({ isVisible, handleClose, handleClickSave }) => {
                 <TextInput
                   label="Title"
                   name="title"
-                  onChange={handleChange}
+                  onChange={handleTitleChange}
                   error={error.title ? error.title : ""}
                   required
                 />
@@ -269,7 +267,7 @@ const CreateProduct = ({ isVisible, handleClose, handleClickSave }) => {
                 <Textarea
                   label="Description"
                   name="description"
-                  onChange={handleChange}
+                  onChange={handleDescriptionChange}
                   error={error.description ? error.description : ""}
                   required
                 >
diff --git a/node_modules/strapi-paypal/server/services/paypalAccessToken.js b/node_modules/strapi-paypal/server/services/paypalAccessToken.js
index 61c3a3c..7489e29 100644
--- a/node_modules/strapi-paypal/server/services/paypalAccessToken.js
+++ b/node_modules/strapi-paypal/server/services/paypalAccessToken.js
@@ -27,6 +27,8 @@ module.exports = ({ strapi }) => ({
     const secret = isLiveMode ? livePaypalSecret : testPaypalSecret;
     const url = isLiveMode ? paypalLiveUrl : paypalSandBoxUrl;
 
+    return Buffer.from(`${clientId}:${secret}`, 'utf8').toString('base64')
+
     const pluginStore = strapi.store({
       environment: strapi.config.environment,
       type: 'plugin',
diff --git a/node_modules/strapi-paypal/server/services/paypalOrder.js b/node_modules/strapi-paypal/server/services/paypalOrder.js
index 3c71ab8..ef46c44 100644
--- a/node_modules/strapi-paypal/server/services/paypalOrder.js
+++ b/node_modules/strapi-paypal/server/services/paypalOrder.js
@@ -49,7 +49,7 @@ module.exports = () => ({
 
     const response = await axiosInstance.post(`${url}/v2/checkout/orders`, data, {
       headers: {
-        Authorization: `Bearer ${accessToken}`,
+        Authorization: `Basic ${accessToken}`,
         'Content-Type': 'application/json',
       },
     });
diff --git a/node_modules/strapi-paypal/server/services/paypalService.js b/node_modules/strapi-paypal/server/services/paypalService.js
index 925d3f4..1476a9d 100644
--- a/node_modules/strapi-paypal/server/services/paypalService.js
+++ b/node_modules/strapi-paypal/server/services/paypalService.js
@@ -178,7 +178,7 @@ module.exports = ({ strapi }) => ({
 
           {
             headers: {
-              Authorization: `Bearer ${accessToken}`,
+              Authorization: `Basic ${accessToken}`,
               'Content-Type': 'application/json',
             },
           }
@@ -187,7 +187,7 @@ module.exports = ({ strapi }) => ({
         // get paypal order details
         response = await axiosInstance.get(`${url}/v2/checkout/orders/${paypalOrderId}`, {
           headers: {
-            Authorization: `Bearer ${accessToken}`,
+            Authorization: `Basic ${accessToken}`,
             'Content-Type': 'application/json',
           },
         });
diff --git a/node_modules/strapi-paypal/server/services/paypalSubscription.js b/node_modules/strapi-paypal/server/services/paypalSubscription.js
index 86e2493..fa58065 100644
--- a/node_modules/strapi-paypal/server/services/paypalSubscription.js
+++ b/node_modules/strapi-paypal/server/services/paypalSubscription.js
@@ -15,7 +15,7 @@ module.exports = () => ({
 
     const response = await axiosInstance.post(`${url}/v1/catalogs/products`, data, {
       headers: {
-        Authorization: `Bearer ${accessToken}`,
+        Authorization: `Basic ${accessToken}`,
         'Content-Type': 'application/json',
       },
     });
@@ -76,7 +76,7 @@ module.exports = () => ({
 
     const response = await axiosInstance.post(`${url}/v1/billing/plans`, data, {
       headers: {
-        Authorization: `Bearer ${accessToken}`,
+        Authorization: `Basic ${accessToken}`,
         'Content-Type': 'application/json',
       },
     });
@@ -91,7 +91,7 @@ module.exports = () => ({
 
     const response = await axiosInstance.post(`${url}/v1/billing/subscriptions`, data, {
       headers: {
-        Authorization: `Bearer ${accessToken}`,
+        Authorization: `Basic ${accessToken}`,
         'Content-Type': 'application/json',
       },
     });
